install.hs

```diff
@@ -163,15 +163,15 @@ updateSubmodules = do
 
 validateCabalNewInstallIsSupported :: Action ()
 validateCabalNewInstallIsSupported = when isWindowsSystem $ do
-  liftIO $ putStrLn $ embedInStars cabalInstallNotSuported
-  error cabalInstallNotSuported
+  liftIO $ putStrLn $ embedInStars cabalInstallNotSuportedFailMsg
```

Comment:
We have to actually add a TODO here, since in upstream cabal, the PR has been merged to support `new-install` on windows, see [#5837](https://github.com/haskell/cabal/issues/5837)

Reply:
todo added